extras/AFC.py

```diff
@@ -1067,23 +1065,35 @@ def get_status(self, eventtime):
         str["system"]['num_extruders'] = len(self.extruders)
         str["system"]["extruders"]={}
 
-        for EXTRUDE in self.extruders.keys():
-            str["system"]["extruders"][EXTRUDE]={}
-            CUR_EXTRUDER = self.printer.lookup_object('AFC_extruder ' + EXTRUDE)
-            str["system"]["extruders"][EXTRUDE]['lane_loaded'] = CUR_EXTRUDER.lane_loaded
-            if CUR_EXTRUDER.tool_start == "buffer":
-                if CUR_EXTRUDER.lane_loaded == '':
-                    str ["system"]["extruders"][EXTRUDE]['tool_start_sensor'] = False
+        for EXTRUDE in self.printer.objects:
```

Comment:
Why did this move away from `self.extruder.keys()`? Iterating over all the printer object every time is going to be costly, compared to an array that is going to be smaller.

Reply:
The afc get_status going away soon as mainsail adjusted